Rearrange sections and other changes. (#106)
xKDR/Survey.jl: docs/src/api.md

# API

## Index

```@index
Module = [Survey]
Order = [:type, :function]
Private = false
```
Survey data can be loaded from a `DataFrame` into a survey design object. The package currently supports simple random sample and stratified sample designs. 
```@docs
AbstractSurveyDesign
SimpleRandomSample
StratifiedSample
```

```@docs
load_data
Survey.mean(x::Symbol, design::SimpleRandomSample)
total(x::Symbol, design::SimpleRandomSample)
quantile
```

> It is often required to estimate population parameters for sub-populations of interest. For example, you make have of heights of people, but you want the average height of male and female separately. 

Reviewer: The last phrase has some errors. Perhaps something like "you may have the height of each subject and you want to find the average height of males and females, respectively." is better?

Author: ```suggestion
It is often required to estimate population parameters for sub-populations of interest. For example, you may have a sample of heights of persons, but you want the average height of male and female separately. 
```

Author: How does the suggestion sound?

Author: Can you suggest the entire paragraph?

Reviewer: I did, it's the one I wrote right before the comment.
